### PR TITLE
Use PHP 7.4 for unit tests

### PR DESCRIPTION
**Pull request by mukeshpanchal27**
## Summary
Resolve unit test failures that occurred after the release of PHP 8.2. 

## Checklist

- [x] PR has either `[Focus]` or `Infrastructure` label.
- [x] PR has a `[Type]` label.
- [x] PR has a milestone or the `no milestone` label.

### CODE REVIEW

**Review by mehulkaklotar — APPROVED** [review]
@mukeshpanchal27 Looks good to me! 🚀

File: .github/workflows/php-test.yml [hunk 1]
```diff
@@ -44,6 +44,8 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: npm
+      - name: Composer update
+        run: composer update
```
Comment (felixarntz) [inline .github/workflows/php-test.yml:54]: Why is this addition needed?

**Review by felixarntz — COMMENTED** [review]
Thanks @mukeshpanchal27, fix looks good, though I have a question on the one change.

File: .github/workflows/php-test.yml (hunk 1 above)
Comment (mukeshpanchal27) [reply .github/workflows/php-test.yml:54]: Thanks, @felixarntz for the review and question.

Without `composer update` it gives the same error [shown here](https://github.com/WordPress/performance/actions/runs/3653782252/jobs/6173598812) as it did not pull updated composer packages.

similar issue replicated on local systems:
1. git clone git@github.com:WordPress/performance.git
2. git checkout fix/unit-tests-failure
3. npm run test-php

If we run `composer update` before step 3, then unit testing works fine.

Comment (felixarntz) [reply .github/workflows/php-test.yml:54]: Thanks @mukeshpanchal27! I still gave this another try, I was thinking maybe it's due to some cache of Composer dependencies. I'm still a bit confused why it is necessary, but you're right that it seems needed for the tests to pass. Let's leave it as is for now 👍

**Review by felixarntz — APPROVED** [review]
Thanks @mukeshpanchal27!

I pushed one more tiny commit https://github.com/WordPress/performance/pull/591/commits/bc9e2dc35fa872bc22ffcde0e18ac591f78736ee to ensure that unit tests also run in the future when making changes to files like `.wp-env.json` since it affects the Docker container we run the logic in.

Btw I notice we get warnings from PHPUnit, opened #594 to fix that.
